src/utilities/cnpj/index.ts

```diff
@@ -96,10 +132,15 @@ export function isValidChecksum(cnpj: string): boolean {
   });
 }
 
-export function isValid(cnpj: string): boolean {
+export interface isValidCnpjOptions {
+  // botar um type Vesions
+  version?: CnpjVersions;
+}
+
+export function isValid(cnpj: string, options: isValidCnpjOptions = {}): boolean {
   if (!cnpj || typeof cnpj !== 'string') return false;
 
-  const numbers = onlyNumbers(cnpj);
+  const validValue = options.version === '2' ? onlyValidCNPJAlphanumeric(cnpj) : onlyNumbers(cnpj);
```

Comment:
```suggestion
  const validValue = options.version == 2 ? onlyValidCNPJAlphanumeric(cnpj) : onlyNumbers(cnpj);
```
dessa forma se passar 2 number ou string vai funcionar igual